### DISCUSSION TITLE
Ethereum

### DISCUSSION THREAD

**Discussion by mohoksoft** [btcpayserver/btcpayserver#4088] [General]
Is there an Ethereum implementation that has been tested and validated?

**Comment by Ou7law007** [comment]
I second this

**Comment by Ou7law007** [comment]
There was an Ethereum implementation in the ancient days
https://ethereum.stackexchange.com/questions/94238/is-there-btcpay-server-for-ethereum

I also wonder why Litecoin is supported in the test mode but not in production.
